src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -80,15 +84,42 @@ const QuickMarcEditor = ({
     ? pristine || submitting
     : submitting;
 
-  const confirmSubmit = useCallback((e) => {
+  const redirectToVersion = useCallback((updatedVersion) => {
```

Comment:
I think it'll be better to use `history.replace` in this case. If a user saves the record several times, the only difference between urls will only be this `relatedRecordVersion` parameter - and when user clicks on browser back button they will be returning to a url with old version and not the Inventory search page. It'll be a smoother experience if you only need to click on back button once
Also, only including `relatedRecordVersion` in `search` removes all other url parameters. Let's use `URLSearchParams` to only change this one parameter and keep everything else

Reply:
Yeah, its much better now